### PR TITLE
Use the raven client to captureException on the 500 error page.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: cliquet/views/errors.py [hunk 1]
```diff
@@ -82,6 +82,10 @@ def error(context, request):
         logger.critical(context.original, exc_info=True)
         return service_unavailable(context, request)
 
+    if hasattr(request, 'raven'):
+        ident = request.raven.get_ident(request.raven.captureException())
+        print("Exception captured in event number: %s" % ident)
```
Comment (leplatrem) [inline cliquet/views/errors.py]: with this we won't catch storage backend errors (ex. DB constraints etc.)

Comment (leplatrem) [inline cliquet/views/errors.py]: instead of print(), use logger.info() ?

File: cliquet/__init__.py [hunk 2]
```diff
@@ -214,13 +214,19 @@ def eos_tween(request):
...
-            settings['cliquet.sentry_url'],
+            dsn=settings['cliquet.sentry_dsn'],
             include_paths=['cornice', 'cliquet'] + extra_projects,
             release=settings['cliquet.project_version'])
 
+        def on_new_request(event):
+            # Attach objects on requests for easier access.
+            event.request.raven = raven_client
+
+        config.add_subscriber(on_new_request, NewRequest)
```
Comment (leplatrem) [inline cliquet/__init__.py]: Add a small test in test_initialization please

File: cliquet/__init__.py [hunk 3]
```diff
@@ -71,7 +71,7 @@
     'cliquet.storage_pool_maxconn': 50,
     'cliquet.storage_max_fetch_size': 10000,
     'cliquet.userid_hmac_secret': '',
-    'cliquet.sentry_url': None,
+    'cliquet.sentry_dsn': None,
```
Comment (leplatrem) [inline cliquet/__init__.py]: update docs then

(I personnally prefer _url everywhere)

File: cliquet/views/errors.py (hunk 1 above)
Comment (Natim) [inline cliquet/views/errors.py]: Actually we catch everything that is raised in a view, since it is a webserver we should catch everything that is not related to server startup.

Comment (leplatrem) [inline cliquet/views/errors.py]: If a DB constraint is violated, it will be catched in the backend and a BackendError will be raised. And sentry will not be aware of it, because it is returned 4 LOC above

Comment (Natim) [inline cliquet/views/errors.py]: Ok so I can leave this but put it 4 LOC before is that right?

Comment (Natim) [inline cliquet/views/errors.py]: I think it would be much better to actually configure it with: http://raven.readthedocs.org/en/latest/integrations/logging.html or http://raven.readthedocs.org/en/latest/integrations/pyramid.html

I will rework that.

**Comment by leplatrem** [comment]
You left it on purpose in monitoring ?


Otherwise, the documentation should probably be updated too :)

**Comment by leplatrem** [comment]
Before merge : 
* remove SentryConfigurationTest``
* add a logger.info("...Application starting...") like before with raven client
* make sure we want to leave sentry in setup.py for monitoring deps
